vic/drivers/cesm/include/vic_driver_cesm.h

```diff
@@ -0,0 +1,192 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * Header file for CESM-VIC routines
+ *
+ * @section LICENSE
+ *
+ * The Variable Infiltration Capacity (VIC) macroscale hydrological model
+ * Copyright (C) 2014 The Land Surface Hydrology Group, Department of Civil
+ * and Environmental Engineering, University of Washington.
+ *
+ * The VIC model is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *****************************************************************************/
+
+#ifndef VIC_DRIVER_CESM_H
+#define VIC_DRIVER_CESM_H
+
+#include <vic_driver_shared_image.h>
+
+#define VIC_DRIVER "CESM"
+
+#define MAXDIMS 10
+#define GLOBALPARAM "vic.globalconfig.txt"
```

Comment:
Does that need to be hardwired? If so clarify that in a comment. Same for next one.